src/graph.cc

```diff
@@ -406,16 +406,16 @@ std::string Edge::GetBinding(const std::string& key) const {
   return env.LookupVariable(key);
 }
 
-bool Edge::GetBindingBool(const string& key) {
+bool Edge::GetBindingBool(const string& key) const {
   return !GetBinding(key).empty();
```

Comment:
Can this function be changed to accept the "StringPiece" class instead?

Reply:
I don't think so: `GetBinding()` internally passes a string reference over to `EdgeEnv::LookupVariable()`, which in turn exploits string API, e.g. it compares the string with other strings.

Reply:
Ah.

Guess we'll need to wait until we have access to std::less<> then.